src/ReactiveUI.Avalonia/AppBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using Avalonia.Controls;
+using Avalonia.Threading;
+using ReactiveUI;
+using Splat;
+
+namespace ReactiveUI.Avalonia
+{
+    public static class AppBuilderExtensions
+    {
+        public static TAppBuilder UseReactiveUI<TAppBuilder>(this TAppBuilder builder)
```

Comment:
I suspect you can eliminate this by adding a IWantsToRegisterStuff() btw. 

To access as discussed in Slack you just need to add a entry to src/ReactiveUI/Properties/AssemblyInfo.cs file to this new assembly.